BroFrame.cpp

```diff
@@ -946,6 +946,40 @@ void CBrowserFrame::CreateRebars()
 			}
 		}
 	}
+	else
+	{
+		// Even though IsRebar option is disabled (EnableRebar=0), create rebar as a
+		// container for only packing cTabWnd purpose.
+		// It fixes the issue (When rebar is disabled, the top of website content 
+		// in tab is hidden unexpectedly, in other words,
+		// the content rendering is clipped and can't access hidden parts)
+		if (m_cTabWnd)
+		{
+			if (m_pwndReBar == NULL)
+			{
+				PROC_TIME_S(CreateRebars_REBAR_NEW_AsContainer)
+				m_pwndReBar = new CMyReBar;
+				m_pwndReBar->m_bMultiThreaded = TRUE;
+				PROC_TIME_E(CreateRebars_REBAR_NEW_AsContainer)
+			}
+			if (!m_pwndReBarCreateFlg)

```

Comment:
I guess that `m_pwndReBarCreateFlg` is `false` when `m_pwndReBar` is NULL, so `m_pwndReBarCreateFlg` is always `false` at this line.
Can we remove this `if` check?

Reply:
Thanks, it should be.